content/applications/services/helpdesk/overview/getting_started.rst

```diff
@@ -2,119 +2,152 @@
+are organized in customizable stages that enable teams to track, prioritize, and solve customer
+issues quickly and efficiently.
 
-Create Helpdesk teams
-=====================
+Create a Helpdesk team
+======================
 
-Setting up multiple teams allows for tickets to be grouped by location or by support type.
-
-To view or modify *Helpdesk* teams, go to :menuselection:`Helpdesk --> Configuration --> Teams`. To
-create a new team, click on the :guilabel:`New` button in the top left of the dashboard. From there,
-name the new team, and fill out the remaining fields as defined in the following sections on the
-form.
+To view or modify *Helpdesk* teams, go to :menuselection:`Helpdesk app --> Configuration -->
+Helpdesk Teams`. To create a new team, click on the :guilabel:`New` button in the top-left of the
+dashboard.
 
 .. image:: getting_started/helpdesk-teams-list.png
    :align: center
-   :alt: View of the Helpdesk teams page in Odoo Helpdesk
+   :alt: View of the Helpdesk teams page in Odoo Helpdesk.
+
+Enter a :guilabel:`Name` for the new team. Enter a description of the team to the field below the
+team name, if desired.
+
+.. important::
+   This description is published on the public facing :doc:`website form <receiving_tickets>` where
+   customers and portal users submit tickets. The description included in this field should **not**
+   include any information that is for internal use only.
 
-Assignment & Visibility
+   .. image:: getting_started/team-description-webform.png
+      :align: center
+      :alt: View of a Helpdesk team's webform displaying the team description.
+
+Visibility & Assignment
 -----------------------
 
+*Visibility* refers to what internal users and portal users will have access to this team and its
+tickets. *Assignment* refers to how users will be assigned to handle each ticket.
+
 Determine to whom the team will be visible
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Under the :guilabel:`Visibility` section, determine who can view this team and its tickets.
 
-- :guilabel:`Invited internal users` have access to the team and tickets they are following. This
-  can be modified on each individual ticket.
-- :guilabel:`All internal users` have access to the team and all of its tickets without being a
-  follower.
-- :guilabel:`Invited portal users and all internal users` have access to the team without being a
-  follower. Portal users will only be able to access tickets that they are following.
+- :guilabel:`Invited internal users (private)` have access to the team and tickets they are
+  following. This can be modified on each individual ticket.
+- :guilabel:`All internal users (company)` have access to the team and all of its tickets without
+  being a follower.
+- :guilabel:`Invited portal users and all internal users (public)` have access to the team without
+  being a follower. Portal users will only be able to access tickets that they are following.
```

Comment:
```suggestion
- :guilabel:`Invited internal users (private)`: only the specific individuals on the team and followers of a specific ticket have access. This can be modified on each individual ticket. No portal users have access to tickets or the team.
- :guilabel:`All internal users (company)`: all users in the company have access to the team and all of its tickets without needing to be a follower on any tickets.
- :guilabel:`Invited portal users and all internal users (public)`: portal users following specific tickets only have access to those specific tickets.  In addition, all users in the company have access to the team and all of its tickets without needing to be a follower.
```

I would add some more information to clarify these selections. Since they are read as sentences, for the last one, I would lead with the invited portal users since the selection  starts with that, to make it less confusing.  No need to take my edits, just an example. (And you can disregard if you disagree, of course).